WaterNetworkManagement/Simulation/SimulationScenario/doc/spec.md

```diff
@@ -0,0 +1,229 @@
+# Simulation Scenario
+
+## Description
+This entity contains an harmonised description of a generic simulation scenario made for the Water Network Management domain. This entity is primarily associated with the water network management vertical and related IoT applications.
+
+## Data Model
+
+A JSON Schema corresponding to this data model can be found [here](../schema.json)
+
+### NGSI-LD common Properties
+-   `id`: Unique identifier.
+
+-   `type`: Entity type. It must be equal to `SimulationScenarios`.
+
+### Simulation Scenario Entity Properties
+
```

Comment:
ok

Reply:
I will add it at the same time as adding descriptions and examples.

Reply:
Just to avoid conflicts issues I'm reviwing your comments, I will commit these modifications. We merge the PR to master and then add the description and exmaples

Reply:
OK